src/main/java/com/google/sps/model/GmailClient.java

```diff
@@ -76,20 +76,22 @@
    *     ignore filter
    * @param emailAgeUnits "d" for days, "h" for hours, "" for ignore email
    * @param unreadOnly true if only returning unread emails, false otherwise
+   * @param isImportant true if only returning important emails, false otherwise
    * @param from email address of the sender. "" if not specified
    * @return string to use in gmail (either client or API) to find emails that match criteria
    */
   static String emailQueryString(
-      int emailAge, String emailAgeUnits, boolean unreadOnly, String from) {
+      int emailAge, String emailAgeUnits, boolean unreadOnly, boolean isImportant, String from) {
```

Comment:
nit: I should've mentioned this in the PR this was introduced. Use `StringBuilder` rather than [creating a new string](https://stackoverflow.com/questions/5234147/why-stringbuilder-when-there-is-string) for each parameter.

Reply:
Fixed this in a separate PR (#92). Thanks!

Reply:
@antmar Good suggestion! I'll use String.join and when completing the finishing touches I'll look at using a builder pattern. I logged an issue for this (#100)